docs/tools/vscode-piral.md

```diff
@@ -1,23 +1,69 @@
 ---
-title: Piral VSCode
+title: Piral VS Code
 ---
 
-# Piral Visual Studio Code Extension
+# Piral VS Code
 
-Piral comes with a Visual Studio Code extension that's available at the official [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=smapiot.vscode-piral).
+## Introduction
+
+[Piral VS Code](https://github.com/smapiot/vscode-piral) is an extension for [Visual Studio Code](https://code.visualstudio.com/). It provides many utilities for developing Piral applications, directly integrated into the editor.
+
+You can install the extension from:
+
+[![VS Code Logo](../diagrams/vscode-vscode-logo.svg){style="background-color: transparent"} **VS Code Marketplace**](https://marketplace.visualstudio.com/items?itemName=smapiot.vscode-piral) &nbsp; &middot; &nbsp;
+[![VSX Logo](../diagrams/vscode-vsx-logo.svg){style="background-color: transparent"} **Open VSX**](https://open-vsx.org/extension/smapiot/vscode-piral)
```

Comment:
Maybe add a sentence that the Open VSX marketplace is used in VS Code Compatible Editors such as Gitpod, Eclipse Theia, or VSCodium.